### DISCUSSION TITLE
Connecting to my Local Jellyfin Server

### DISCUSSION THREAD

**Discussion by BMBT-DK** [CyferShepard/Jellystat#104] [Q&A]
Hello,

This is probably a dumb question, but I ran into an issue in terms of conencting my Jellystat to Jellyfin.
So i've run the docker and it seems to succesfully made the jellystat, so when I open it, and attempt to connect them I get an auth error.
I just made a URL in the dashboard to test the connection and API key with it. Beyond that I have changed no settings in Jellyfin, what could the issue be?

![image](https://github.com/CyferShepard/Jellystat/assets/143399881/61d6ceac-f9a4-4a3f-8729-b47aa083503c)

**Comment by CyferShepard** [comment]
hi @BMBT-DK , just double check the api key from jellyfin, also confirm if you can access jellyfin if you visit the same url provided, 403 usually means access was forbidden

**Reply by BMBT-DK** [reply]
The APIKEY should be correct, I've tried to remove the base url.
But here comes another stupid question from that, as I try to connect to the localhost that connects me it gives new error that I can't connect
![image](https://github.com/CyferShepard/Jellystat/assets/143399881/28f7b783-de8f-433f-be03-3a9bb8e96746)

**Reply by CyferShepard** [reply]
so this issue may be due to how your docker network is setup, if its in bridge mode this wont work as the term localhost will refer to the ip of the docker container and not your host machine. Your better of using your host machines IP instead

**Reply by BMBT-DK** [reply]
The docker settings is just set to default like your docker page set.
But yea now it atleast attempts to connect but gives another issue
![image](https://github.com/CyferShepard/Jellystat/assets/143399881/14e26ecf-3b5b-42b2-9f1a-50361bf80288)

**Reply by CyferShepard** [reply]
EAI means the DNS name didn't resolve, you'd need to enter the IP address of your host machine and not the word host machine unless youv got that setup in a DNS resolver like Pi-Hole

**Reply by BMBT-DK** [reply]
Seems I may not have set it up correctly, as it cannot find the URL, and neither can i on my machine, I just replaced Hostmachine with the IP4 address otherwiese the URL is the same as last time. 
![image](https://github.com/CyferShepard/Jellystat/assets/143399881/c6ccf81d-b7d8-47a5-aa8f-634c031c7bfc)
